controllers/front/GuestTrackingController.php

```diff
@@ -91,10 +92,16 @@ public function postProcess()
                     [],
                     'Shop.Forms.Help'
                 );
-            } elseif (strlen($password) < Validate::PASSWORD_LENGTH) {
+            } elseif (!Validate::isAcceptablePasswordLength($password)) {
                 $this->errors[] = $this->trans(
-                    'Your password must be at least %min% characters long.',
-                    ['%min%' => Validate::PASSWORD_LENGTH],
+                    'Your password length must be between %s and %s',
```

Comment:
```suggestion
                    'Your password length must be between %d and %d',
```

You're the best @Julievrz